src/uwtools/resources/jsonschema/chgres-cube.jsonschema

```diff
@@ -92,6 +85,7 @@
                     },
                     "external_model": {
                       "enum": [
+                        "FV3GFS",
```

Comment:
This is not a valid recognized option for chgres_cube. I do not know why SRW is using this value. There is also an instance of `external_model = "UFS-CASE-STUDY"` which is not a recognized value, either.

I don't know how to rectify this other than to remove the enum requirement from this entry altogether.

Reply:
Ugh - I'd love to try to change it on the SRW side but that'll certainly break far too much. Reluctantly, I have to agree